src/buffer.rs

```diff
@@ -531,11 +531,9 @@ impl Debug for Buffer {
     /// Writes a debug representation of the buffer to the given formatter.
     ///
     /// The format is like a pretty printed struct, with the following fields:
-    /// area: displayed as Rect { x: 1, y: 2, width: 3, height: 4 }
-    /// content: displayed as a list of strings representing the content of the
-    ///     buffer
-    /// styles: displayed as a list of
-    ///     { x: 1, y: 2, fg: Color::Red, bg: Color::Blue, modifier: Modifier::BOLD }
+    /// * `area`: displayed as `Rect { x: 1, y: 2, width: 3, height: 4 }`
+    /// * `content`: displayed as a list of strings representing the content of the buffer
+    /// * `styles`: displayed as a list of `{ x: 1, y: 2, fg: Color::Red, bg: Color::Blue, modifier: Modifier::BOLD }`
```

Comment:
I'd suggest leaving this as a 100 char line limit (that limit is used quite a few places in the rust ecosystem). Objectively it helps to avoid horizontal scrolling or wrapping being necessary to read code:

E.g.:
<img width="1440" alt="image" src="https://github.com/tui-rs-revival/ratatui/assets/381361/ac6554a2-112f-45e1-aa57-ecae51221d3f">

vs current:

<img width="1440" alt="image" src="https://github.com/tui-rs-revival/ratatui/assets/381361/82a323c1-d252-42e8-8da9-b5e7ec221ad8">